app/bundles/CampaignBundle/Entity/CampaignRepository.php

```diff
@@ -641,7 +641,12 @@ public function getCampaignsSegmentShare($segmentId, $campaignIds = []): array
      * or empty array if nothing found.
      *
      * @param int $id
+     *
+     * @deprecated The method is deprecated and will be removed in Mautic 8.x.
+     * Use `\Mautic\CampaignBundle\Entity\EventRepository::getCampaignEmailEvents` instead.
```

Comment:
polish: Minor change

```suggestion
     * Use the `\Mautic\CampaignBundle\Entity\EventRepository::getCampaignEmailEvents()` method instead.
```